pkg/controller/util.go

```diff
@@ -2,22 +2,23 @@ package controller
 
 import (
 	"fmt"
+	"math/rand"
+	"strings"
 	"time"
 	"github.com/golang/glog"
-	. "github.com/kubevirt/containerized-data-importer/pkg/common"
 	"github.com/pkg/errors"
 	"k8s.io/api/core/v1"
 	apierrs "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
-	"strings"
+	. "kubevirt.io/containerized-data-importer/pkg/common"
 )
 
 const DataVolName = "cdi-data-vol"
 const ImagePathName = "image-path"
```

Comment:
this seems like an non-intuitive name for a volume. It seems more like the name of the path to the cloner image. Can the volume name be something like _clone-src-vol_, and _clone-tgt-vol_?